diff --git a/xpcom/io/nsAppFileLocationProvider.cpp b/xpcom/io/nsAppFileLocationProvider.cpp
--- a/xpcom/io/nsAppFileLocationProvider.cpp
+++ b/xpcom/io/nsAppFileLocationProvider.cpp
@@ -178,13 +178,7 @@ nsAppFileLocationProvider::GetFile(const
   else if (nsCRT::strcmp(aProp, NS_SYSTEM_PLUGINS_DIR) == 0) {
 #ifdef ENABLE_SYSTEM_EXTENSION_DIRS
     static const char* const sysLPlgDir =
-#if defined(HAVE_USR_LIB64_DIR) && defined(__LP64__)
-      "/usr/lib64/mozilla/plugins";
-#elif defined(__OpenBSD__) || defined (__FreeBSD__)
-      "/usr/local/lib/mozilla/plugins";
-#else
-      "/usr/lib/mozilla/plugins";
-#endif
+      "/app/lib/mozilla/plugins";
     rv = NS_NewNativeLocalFile(nsDependentCString(sysLPlgDir),
                                false, getter_AddRefs(localFile));
 #else
